main.lua

```diff
@@ -663,7 +663,7 @@ local function DispatchText(guid, event, value, spellid, spellicon, crit, miss,
         guid = guid,
         icon = spellicon,
         text = text,
-        value = value,
+        amount = amount,
```

Comment:
miss info is uhm ... missing here.
has to be provided otherwise when sorting `(s1.miss ~= s2.miss)` is never going to be true
```suggestion
        amount = amount,
        miss = miss,
```